.github/workflows/docs.yml

```diff
@@ -7,6 +7,11 @@ on:
   pull_request:
     branches:
     - master
+    paths-ignore:
```

Comment:
Would it be easier to [include paths](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths) instead of excluding them?

We know for sure we wanna trigger for `docs` changes, then we have a couple of files in dcd/ddev that feed into docs, so we could specify those.

Reply:
I'll open a smarter PR later 🙂